website/docs/reference/snowflake-permissions.md

```diff
@@ -20,4 +20,5 @@ grant select on future views in database database_name to role role_name;
 grant usage on all schemas in database database_name to role role_name;
 grant select on all tables in database database_name to role role_name;
 grant select on all views in database database_name to role role_name;
-```
\ No newline at end of file
+```
+Disclaimer
```

Comment:
Was your intent to include more text after "Disclaimer"?

Here's the way it is rendering for me:

<img width="673" alt="image" src="https://user-images.githubusercontent.com/44704949/189991645-c8e5da69-dbb1-455a-9018-ee8c04a930f5.png">

Reply:
Ha! Wow, I need to close my laptop. Removed in [3555772](https://github.com/dbt-labs/docs.getdbt.com/pull/2002/commits/3555772830af117c8757a4989283074f914c10fa)